plugins/scaffolder-backend/src/scaffolder/actions/builtin/helpers.ts

```diff
@@ -296,3 +296,7 @@ export function getGitCommitMessage(
     ? gitCommitMessage
     : config.getOptionalString('scaffolder.defaultCommitMessage');
 }
+
+export function familiarizeEntityName(name: string): string {
```

Comment:
Think we should call this `entityRefToName` or something instead.

Reply:
Renamed.